src/main/scala/com/microsoft/hyperspace/index/IndexCollectionManager.scala

```diff
@@ -168,7 +168,7 @@ private[hyperspace] object IndexSummary {
       entry.numBuckets,
       entry.derivedDataset.properties.schemaString,
       entry.content.root,
-      entry.plan(spark).toString,
+      entry.derivedDataset.properties.schemaString, // TODO
```

Comment:
@imback82 which property would be here? or just remove "queryPlan"?
``` * TODO: Finalize about adding these: data location, signatures, file lists etc.```

Reply:
I think we can just remove it for now (and add additional info based on feedback)

Reply:
ok I'll remove this along with the next commit :)

And I think it's better not to remove com/microsoft/hyperspace/index/serde and its test as the test seems useful later ..?

Reply:
yea, we can remove it as a separate PR if needed.